### PR TITLE
Kat Fries: First commit is very late

### PR DESCRIPTION
**Pull request by cpoIT**
First commit.

### CODE REVIEW

**Comment by cpoIT** [comment]
MVP done

**Comment by cpoIT** [comment]
mistake fixed

**Comment by cpoIT** [comment]
more stretch

File: js/index.js [hunk 1]
```diff
@@ -38,5 +38,198 @@ const siteContent = {
...
+    if (arrNumAndVal[0][i] === i) {
+      orderedTags.push(arrNumAndVal[1][i]);
+    }
+  }
+  return orderedTags;
+}
+let aTagsArr = orderedTagNames(navTags);
+
+
+function createATags (arr) {
```
Comment (CookieMonsta89) [inline js/index.js]: Great use of functions in your assignment.  Here you could have used a forEach instead of the for loop.

File: js/index.js [hunk 2]
```diff
@@ -38,5 +38,198 @@ const siteContent = {
...
+newA2.href = "#";
+newA2.style.color = 'green';
+
+// nav.appendChild("Blog");
+
+// var p = document.createElement("p");
+// document.body.appendChild(p);
+
+
+aTags.forEach(tag => tag.style.color = 'green');
```
Comment (CookieMonsta89) [inline js/index.js]: Great use of the forEach instead of the forLoop.

File: js/index.js [hunk 3]
```diff
@@ -38,5 +38,198 @@ const siteContent = {
...
+nav.setAttribute("style", "color: green");
+nav.style.color = 'green';
+
+ctaTextButton.setAttribute("style", "color: cornflowerBlue; font-weight: bold; background-color: whitesmoke; border: 1px solid cornflowerBlue");
+
+ctaTextButton.addEventListener("mouseover", function( event ) {
```
Comment (CookieMonsta89) [inline js/index.js]: Nice job creating event listeners.  Keep up the good work

**Pull request closed without merging**